scripts/plugins/openvswitch-config-update

```diff
@@ -94,10 +90,7 @@ def update(session, args):
         new_controller = True
         ret_str += "Successfully set controller to %s.  " % controller
 
-    try:
-        pool_fail_mode = pool["other_config"]["vswitch-controller-fail-mode"]
-    except KeyError as e:
-        pool_fail_mode = None
+    pool_fail_mode = pool["other_config"].get("vswitch-controller-fail-mode")
```

Comment:
I don't think this is equivalent code, fetching other-config can also raise a KeyError